examples/Thermostat/Client/Program.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc;
+using Igloo;
+using Microsoft.Extensions.Logging;
+using System.CommandLine;
+
+using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+    builder
+        .AddSimpleConsole()
+        .AddFilter("IceRpc", LogLevel.Information));
+
+// Create a client connection to the cloud server.
```

Comment:
```suggestion
// Create a client connection to the "cloud" server.
```

To avoid confusion I would always put double quoutes around cloud.

Reply:
Removed cloud. There is no cloud server here.